config.template

```diff
@@ -105,6 +127,7 @@ regionIndicesToPlot = [6]
 N_movavg = 12
 
 [nino34_timeseries]
+
 ## compare to output from another model run?
```

Comment:
Agreed.  I think the comment is helpful, analogous to docstrings for python functions, classes, etc.